src/main/java/me/botsko/prism/utils/block/TagLibrary.java

```diff
@@ -0,0 +1,193 @@
+package me.botsko.prism.utils.block;
+
+import me.botsko.prism.utils.MaterialTag;
+import me.botsko.prism.utils.MaterialTag.MatchMode;
+import org.bukkit.Material;
+import org.bukkit.Tag;
+import org.bukkit.block.data.BlockData;
+
+import java.util.EnumMap;
+import java.util.EnumSet;
+import java.util.Locale;
+
+/**
+ * Created for use for the Add5tar MC Minecraft server
+ * Created by benjamincharlton on 23/05/2020.
+ */
+public class TagLibrary {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'TagLibrary' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5612535)